core/src/main/scala/cats/data/Ior.scala

```diff
@@ -91,6 +96,10 @@ sealed abstract class Ior[+A, +B] extends Product with Serializable {
 
   final def merge[AA >: A](implicit ev: B <:< AA, AA: Semigroup[AA]): AA =
     fold(identity, ev.apply, (a, b) => AA.combine(a, b))
+  final def mergeLeft[AA >: A](implicit ev: B <:< AA): AA =
+    fold(identity, ev.apply, (a, _) => a)
```

Comment:
I don't think you need `ev.apply` here. I *think* `B <:< AA` is already a function `B => AA` so you can pass it directly without allocating another closure.

Although, it would be interesting to know if scalac avoids allocating a closure when you do `fn.apply` and instead just passes `fn`.